db/migrate/20211001151212_update_patient_table_string_lengths.rb

```diff
@@ -0,0 +1,185 @@
+    change_column :patients, :user_defined_id_nndss, :string, limit: 200
+
+    # Address
+    change_column :patients, :address_line_1, :string, limit: 200
+    change_column :patients, :address_line_2, :string, limit: 200
+    change_column :patients, :address_city, :string, limit: 200
+    change_column :patients, :address_state, :string, limit: 200
+    change_column :patients, :address_zip, :string, limit: 200
+    change_column :patients, :address_county, :string, limit: 200
+    change_column :patients, :foreign_address_line_1, :string, limit: 200
+    change_column :patients, :foreign_address_line_2, :string, limit: 200
+    change_column :patients, :foreign_address_city, :string, limit: 200
+    change_column :patients, :foreign_address_country, :string, limit: 200
+    change_column :patients, :foreign_address_zip, :string, limit: 200
+    change_column :patients, :foreign_address_line_3, :string, limit: 200
+    change_column :patients, :foreign_address_state, :string, limit: 200
+    change_column :patients, :monitored_address_line_1, :string, limit: 200
+    change_column :patients, :monitored_address_line_2, :string, limit: 200
+    change_column :patients, :monitored_address_city, :string, limit: 200
+    change_column :patients, :monitored_address_state, :string, limit: 200
+    change_column :patients, :monitored_address_zip, :string, limit: 200
+    change_column :patients, :monitored_address_county, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_line_1, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_line_2, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_city, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_state, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_zip, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_county, :string, limit: 200
+
+    # Contact
+    change_column :patients, :preferred_contact_method, :string, limit: 32
+    change_column :patients, :preferred_contact_time, :string, limit: 16
+    change_column :patients, :primary_telephone, :string, limit: 16
+    change_column :patients, :primary_telephone_type, :string, limit: 16
+    change_column :patients, :secondary_telephone, :string, limit: 16
+    change_column :patients, :secondary_telephone_type, :string, limit: 16
+    change_column :patients, :international_telephone, :string, limit: 50
+    change_column :patients, :email, :string, limit: 200
+
+    # Arrival
+    change_column :patients, :port_of_origin, :string, limit: 200
+    change_column :patients, :flight_or_vessel_number, :string, limit: 200
+    change_column :patients, :flight_or_vessel_carrier, :string, limit: 200
+    change_column :patients, :port_of_entry_into_usa, :string, limit: 200
+    change_column :patients, :source_of_report, :string, limit: 32
```

Comment:
Max for this is 22
`ValidationHelper::VALID_PATIENT_ENUMS[:source_of_report].reject(&:nil?).map(&:length).max`